src/commands/utility/info/category.js

```diff
@@ -0,0 +1,39 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoCategory extends Command {
+  constructor (client) {
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'string',
```

Comment:
Will be better if you use `channel` param and customize it to support categories.